operator/.gitignore

```diff
@@ -0,0 +1 @@
+molecule/maistra-e2e/assets/secret.yaml
```

Comment:
Can you explain what this secret.yaml is? It is being ignored, but is it required to run this test?

Reply:
It is only when you are going to deploy from maistra on quay.
It is possible to deploy from docker.io/maistra ubi images so the secret is not needed.